e2e/app.po.ts

```diff
@@ -1,11 +1,12 @@
 import { browser, element, by } from 'protractor/globals';
 
 export class Ng2JsonEditorPage {
+
   navigateTo() {
     return browser.get('/');
   }
 
-  getParagraphText() {
-    return element(by.css('app-root h1')).getText();
+  getInputElementById(id: string) {
```

Comment:
Either `getTextareaChildrenById` or separate into 2 different methods (one to get element by id and the other one to get textareas inside of that element)